script.simkl/resources/language/English/strings.po

```diff
@@ -0,0 +1,103 @@
+# Kodi Media Center language file
```

Comment:
Please move language file according to following folder structure http://kodi.wiki/view/language_support#Layout_of_the_language_folders